ecs logging can throw an NPE

ecs-logging can throw an NPE, as per stack trace below.
```
Caused by: java.lang.NullPointerException                                                                                                                 [849/1871]        at co.elastic.logging.JsonUtils.quoteAsString(JsonUtils.java:63)

        at co.elastic.logging.EcsJsonSerializer.serializeException(EcsJsonSerializer.java:163)
        at co.elastic.logging.log4j.EcsLayout.format(EcsLayout.java:62)
        at org.apache.log4j.WriterAppender.subAppend(WriterAppender.java:310)
        at org.apache.log4j.DailyRollingFileAppender.subAppend(DailyRollingFileAppender.java:369)
        at org.apache.log4j.WriterAppender.append(WriterAppender.java:162)
        at org.apache.log4j.AppenderSkeleton.doAppend(AppenderSkeleton.java:251)
        at org.apache.log4j.helpers.AppenderAttachableImpl.appendLoopOnAppenders(AppenderAttachableImpl.java:66)
        at org.apache.log4j.Category.callAppenders(Category.java:206)
        at org.apache.log4j.Category.forcedLog(Category.java:391)
        at org.apache.log4j.Category.error(Category.java:322)
        at private code
```

--- a/ecs-logging-core/src/main/java/co/elastic/logging/EcsJsonSerializer.java
+++ b/ecs-logging-core/src/main/java/co/elastic/logging/EcsJsonSerializer.java
@@ -159,9 +159,13 @@ public class EcsJsonSerializer {
             builder.append("\"error.type\":\"");
             JsonUtils.quoteAsString(thrown.getClass().getName(), builder);
             builder.append("\",");
-            builder.append("\"error.message\":\"");
-            JsonUtils.quoteAsString(thrown.getMessage(), builder);
-            builder.append("\",");
+
+            String message = thrown.getMessage();
+            if (message != null) {
+                builder.append("\"error.message\":\"");
+                JsonUtils.quoteAsString(message, builder);
+                builder.append("\",");
+            }
             if (stackTraceAsArray) {
                 builder.append("\"error.stack_trace\":[").append(NEW_LINE);
                 formatThrowableAsArray(builder, thrown);

--- a/ecs-logging-core/src/test/java/co/elastic/logging/AbstractEcsLoggingTest.java
+++ b/ecs-logging-core/src/test/java/co/elastic/logging/AbstractEcsLoggingTest.java
@@ -104,6 +104,13 @@ public abstract class AbstractEcsLoggingTest {
     }
 
     @Test
+    void testLogExceptionNullMessage() throws Exception {
+        error("test", new RuntimeException());
+        assertThat(getLastLogLine().get("error.message")).isNull();
+        assertThat(getLastLogLine().get("error.type").textValue()).isEqualTo(RuntimeException.class.getName());
+    }
+
+    @Test
     void testLogOrigin() throws Exception {
         debug("test");
         assertThat(getLastLogLine().get("log.origin").get("file.name").textValue()).endsWith(".java");

